app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -101,15 +103,16 @@ class ProfileEditFragmentPresenter @Inject constructor(
         ProfileId.newBuilder().setInternalId(internalProfileId).build(),
         binding.profileEditAllowDownloadSwitch.isChecked
       ).toLiveData().observe(
```

Comment:
Could probably fit on one line now for a bit cleaner code, e.g.: ``).toLiveData().observe(activity) {``